WordPress/gutenberg
# RSS: Border & Spacing support

## What?
Adding border support & spacing support for RSS block.

Part of: https://github.com/WordPress/gutenberg/issues/43247
Part of: https://github.com/WordPress/gutenberg/issues/43243


## Why?
RSS block is missing border & spacing support



## How?
Adds the border & spacing support via block.json. 


## Testing Instructions

- Go to Global Styles setting ( under appearance > editor > styles > edit styles > blocks ).
- Make sure that RSS block's border is configurable via Global Styles.
- Verify that Global Styles are applied correctly in the editor and frontend.
- Edit template/page, Add Page List block and apply the border, spacing( padding & margin) styles.
- Verify that block styles take precedence over global styles.
- Verify that block styles display correctly in both the editor and frontend.



## Screenshots or screencast 

Backend:
![rssbackend](https://github.com/user-attachments/assets/2fa81125-0eff-4c00-8bae-ad4b5c350488)

Frontend:
![rssfrontend](https://github.com/user-attachments/assets/c317cc38-3c6c-441c-b5be-f16268b7c6e0)
